src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/ILCompiler.Reflection.ReadyToRun.csproj

```diff
@@ -8,7 +8,7 @@
     <AssemblyKey>Open</AssemblyKey>
     <IsDotNetFrameworkProductAssembly>true</IsDotNetFrameworkProductAssembly>
     <!-- ILSpy requires this assembly to target netstandard2.0 -->
-    <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
Post-merge nit: you can also drop the `s` for a tiny build-perf improvement.

Reply:
Have we ever considered a message when TargetFrameworks contains only one entry? It would be noisy of course if there were conditions involved such that sometimes it has more than one.

Reply:
Run regex find&replace to fixup all places in the repo: https://github.com/dotnet/runtime/pull/66198

Reply:
The change also makes the logs easier to read, as there's no re-invocation of itself.